src/auto/custom_sorter.rs

```diff
@@ -0,0 +1,41 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use glib::object::IsA;
+use glib::translate::*;
+use gtk_sys;
+use std::fmt;
+use Sorter;
+
+glib_wrapper! {
+    pub struct CustomSorter(Object<gtk_sys::GtkCustomSorter, gtk_sys::GtkCustomSorterClass, CustomSorterClass>) @extends Sorter;
+
+    match fn {
+        get_type => || gtk_sys::gtk_custom_sorter_get_type(),
+    }
+}
+
+impl CustomSorter {
+    //pub fn new(sort_func: /*Unimplemented*/Fn(/*Unimplemented*/Option<Fundamental: Pointer>, /*Unimplemented*/Option<Fundamental: Pointer>) -> i32, user_data: /*Unimplemented*/Option<Fundamental: Pointer>) -> CustomSorter {
```

Comment:
This would be nice to implement, just like `gtk_custom_filter_new()` and `gtk_custom_layout_new()` but they probably all need manual code.

Note that the `gpointer` item parameters to these functions are always at least GObject subclasses. That needs some `type GLib.Object` annotations.

Reply:
I have opened issues to track both `CustomSorter` & `CustomLayout` #22 #23